src/nvim/lua/vim.lua

```diff
@@ -9,6 +9,9 @@ end
 
 -- Gets process info from the `ps` command.
 -- Used by nvim_get_proc() as a fallback.
+--!brief Gets the stuff for pid
+--!param pid The pid
+--!return A table
```

Comment:
any idea if there are other syntaxes to choose from, besides `--!` ? (not too important, but I know doxygen has a bunch, would be nice if `--@` worked since that's similar to what we use in the C files.)

Reply:
Yes, very much, that's what I meant by "how should those doc comments look like". I'll make it `--@return` then.